src/scripting/lua_map.cc

```diff
@@ -1063,6 +1085,51 @@ HasInputs
          if b.valid_inputs then b:set_inputs(b.valid_inputs) end
 */
 
+/* RST
+   .. method:: set_priority(ware, prio [, cs_setting = false])
+
+      Sets the priority for the given ware inputqueue.
+
+      :arg ware: ware name
+      :type ware: :class:`string`
+      :arg prio: The new priority. One of `2` (low), `4` (normal), or `8` (high).
```

Comment:
Lua scripters should not be bothered with C++ implementation details. Make these "low", "normal" and "high" and convert them in the function.